lib/trento/domain/cluster/cluster.ex

```diff
@@ -262,6 +267,26 @@ defmodule Trento.Domain.Cluster do
     }
   end
 
+  def execute(
+        %Cluster{cluster_id: cluster_id} = cluster,
+        %DeregisterClusterHost{
+          host_id: host_id,
+          cluster_id: cluster_id
+        } = command
+      ) do
+    cluster
+    |> Multi.new()
+    |> Multi.execute(fn _ ->
+      [
```

Comment:
Do we need to wrap the `HostRemovedFromCluster` event in a list?
I think we could simply send it by itself, right?

Reply:
+1